shell/platform/windows/public/flutter_windows.h

```diff
@@ -20,9 +20,9 @@ extern "C" {
 
 typedef void (*VoidCallback)(void* /* user data */);
 
-// Opaque reference to a Flutter window controller.
-typedef struct FlutterDesktopViewControllerState*
-    FlutterDesktopViewControllerRef;
```

Comment:
Question since this is a public header that is, in rare cases, used directly rather than via the client wrapper: who is this likely to break?

Looking at the current code, I don't actually see anywhere where we've _publicly_ defined `FlutterDesktopViewControllerState` itself, so it _seems_ unlikely that anyone's going to be depending on it other than as an opaque pointer.

Reply:
> who is this likely to break?

The affected users I'm aware of I found are FFI projects:

* Rust: https://github.com/nativeshell/nativeshell/blob/b78fba7ae453fee73ca5666d2175a72d872f77c9/nativeshell/src/shell/platform/win32/flutter_sys.rs#L96-L101
* Dart: https://github.com/dart-windows/win32_runner/blob/45207ec3c7ac4b410b1df0d573e8903ee084ad58/lib/src/ffi.dart#L59
* .NET: https://github.com/LiveOrNot/FlutterSharp/blob/8b24bdf14465c090b53ecc04c0c2c2598ae7aff3/FlutterSharp/Integrations/FlutterInterop.cs#L21

I didn't find any other uses of this type on GitHub and google3 code searches. It seems the impact of this breaking change is limited.

/cc @knopp